pkg/controller/controller.go

```diff
@@ -170,3 +163,24 @@ func (c *PodController) deleteObjects(podObj *v1.Pod, parentJobName string, dryR
 	}
 	return
 }
+
+
+func (c *PodController) getParentJobName(podObj *v1.Pod, version string) (parentJobName string) {
+
+	intVersion, _ := strconv.Atoi(version);
+	if intVersion < 8 {
```

Comment:
I think we should check for both minor and major version here, because it will stop working correctly after k8s v2.0